beets/autotag/__init__.py

```diff
@@ -45,13 +45,13 @@ def apply_item_metadata(item, track_info):
     if track_info.data_source:
         item.data_source = track_info.data_source
 
-    if track_info.lyricist is not None:
+    if track_info.get('lyricist'):
```

Comment:
Good point; sorry I didn't notice that. (Perhaps it would be useful to add a doctoring to that effect?) So maybe we can leave these checks how they are, just to keep a clean diff.